### PR TITLE
octique:0.1.0

### PR DESCRIPTION
**Pull request by 0x6b**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: GitHub [Octicons](https://primer.style/foundations/icons/) for Typst.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

### CODE REVIEW

**Comment by laurmaedje** [comment]
Since the assets aren't necessary for the package (they are inlined), I would rather not have them in this repository. Even though they are excluded from the package, it's a lot of unnecessary files for this repo.

**Comment by laurmaedje** [comment]
For the README, you could maybe use external links to a repository of yours.

**Comment by 0x6b** [comment]
@laurmaedje That makes sense. I've taken your feedback and updated the README accordingly. Thanks!

File: packages/preview/octique/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "octique"
+version = "0.1.0"
+entrypoint = "octique.typ"
+authors = ["0x6b"]
+license = "MIT"
+description = "GitHub [Octicons](https://primer.style/foundations/icons/) for Typst."
```
Comment (laurmaedje) [inline packages/preview/octique/0.1.0/typst.toml:7]: I've missed this the first time around, but the description can only be plain-text, not Markdown.

File: packages/preview/octique/0.1.0/typst.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+homepage = "https://github.com/0x6b/typst-octique"
```
Comment (laurmaedje) [inline packages/preview/octique/0.1.0/typst.toml:8]: This is also redundant.

**Comment by 0x6b** [comment]
@laurmaedje Added two commits to address your point. Thanks!

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**